frontend/ui/data/koptoptions.lua

```diff
@@ -34,7 +34,7 @@ local KoptOptions = {
                 alternate = false,
                 values = {0, 1, 2},
                 default_value = DKOPTREADER_CONFIG_TRIM_PAGE,
-                enabled_func = function() return Device:isTouchDevice() end,
+                enabled_func = Device:isTouchDevice,
```

Comment:
`Module:functionName()` is short for `Module.functionName(self)`. `Module.functionName` is a reference to a function, which when possible is preferable to an anonymous function. `Module:functionName` is not a thing. ;-)

Reply:
PS I just meant put it in the reply, not push it.

Reply:
PPS Could you also change the above to `current_func = Screen.getScreenMode,`?

Reply:
Yes sure will do, once Git start to cooperate ...